File: YSY/11_28/배열 만들기 2.py
def solution(l, r):
    answer = []
    for i in range(l,r+1):
        if set(list(str(i))) == {'5', '0'} or set(list(str(i))) == {'0', '5'} or set(list(str(i))) == {'5'}: 
            answer.append(i)
    if answer:
        answer.sort()
    else:
        answer.append(-1)
    return answer

File: YSY/12_26/배열 만들기 2.cpp
#include <string>
#include <vector>
#include <iostream>

using namespace std;

vector<int> solution(int l, int r) {
    vector<int> answer;
    for (int i=l;i<=r;i++)
    {
        int num=i;
        while(num!=0)
        {
            if (num%10==0 || num%10==5)
                num=num/10;
            else
                break;
        }
        if(num==0)
            answer.push_back(i);
    }
    if (answer.size()==0)
        answer.push_back(-1);
    return answer;
}